### PR TITLE
[7.0.1xx-rc2] Make the vsman's target name unique for this branch.

### PR DESCRIPTION
**Pull request by rolfbjarne**
Also only include platforms we're shipping from this branch.

### CODE REVIEW

File: dotnet/Workloads/vs-workload.template.props [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
   <PropertyGroup>
-    <TargetName>Microsoft.NET.Sdk.MaciOS.Workload.@VSMAN_VERSION@</TargetName>
+    <TargetName>Microsoft.NET.Sdk.MacCatalyst_macOS.net70RC2.Workload.@VSMAN_VERSION@</TargetName>
```
Comment (pjcollins) [inline dotnet/Workloads/vs-workload.template.props:4]: Could we keep this naming consistent with what was done here? https://github.com/xamarin/xamarin-macios/pull/16060/files (e.g. `Microsoft.NET.Sdk.MacCatalyst.macOS.Workload`)

**Review by dalexsoto — APPROVED** [review]

File: dotnet/Workloads/vs-workload.template.props (hunk 1 above)
Comment (rolfbjarne) [reply dotnet/Workloads/vs-workload.template.props]: @pjcollins fixed

Comment (pjcollins) [reply dotnet/Workloads/vs-workload.template.props]: Sorry I should have mentioned that we should keep the version suffix, that change was added to differentiate these manifests from the net6.0 ones that are included side by side.

Comment (rolfbjarne) [reply dotnet/Workloads/vs-workload.template.props]: @pjcollins fixed!

**Review by pjcollins — APPROVED** [review]

**Comment by rolfbjarne** [comment]
Test failures are unrelated (https://github.com/xamarin/maccore/issues/2450).

**Pull request merged**